### ISSUE TITLE
Bug - Incomplete month of August

### ISSUE THREAD

**Issue by juliendu11** [Kapcash/vue-use-calendar#2]
Hi, The library is very good and corresponds a lot to my specifications but I encounter a small problem

When I use the lib with these parameters:

```js
const { useMonthlyCalendar } = useCalendar({
  startOn: "Fri Jul 01 2022 00:00:00 GMT+0200 (heure d’été d’Europe centrale)" => Date Object
  firstDayOfWeek: 1
);
```

And I use `nextMonth()`, days are missing for August, The month starts directly on the 8th day

```

"2022-07-25"
"2022-07-26"
"2022-07-27"
"2022-07-28"
"2022-07-29"
"2022-07-30"
"2022-07-31"
"2022-08-08"
"2022-08-09"
"2022-08-10"
"2022-08-11"
"2022-08-12"
"2022-08-13"
"2022-08-14"
"2022-08-15"
"2022-08-16"
"2022-08-17"
"2022-08-18"
"2022-08-19"
"2022-08-20"
"2022-08-21"
"2022-08-22"
"2022-08-23"
....
```

I don't have this problem when I use with this configuration:

```js
const { useMonthlyCalendar } = useCalendar({
  startOn: "Fri Jul 01 2022 00:00:00 GMT+0200 (heure d’été d’Europe centrale)" => Date Object
  firstDayOfWeek: 0
);
```

So the problem must come from `firstDayOfWeek`, I think starting the week on Monday must mess up the day generation logic

Thanks for advance

**Comment by Kapcash** [comment]
Hi @juliendu11! Thanks for reporting this, I'm glad someone uses the library :)

I just fixed the issues and published version v0.0.2. Let me know if it's better and if you find any other problem.
The library sure isn't perfect yet, but it's getting there!

**Comment by juliendu11** [comment]
Thank you very much, the problem has been fixed! :+1:
